README.md

```diff
@@ -4,7 +4,27 @@
 [![Chat](https://img.shields.io/static/v1.svg?label=chat&message=mattermost&color=145dbf)](https://mattermost.eclipse.org/eclipse/channels/eclipse-codewind)
 
 # Codewind for VS Code
-Create and develop cloud-native, containerized web applications from VS Code.
+Codewind for VS Code provides tools to help build cloud-native, containerized applications from VS Code, regardless of which language you use. Rapidly create an application from a template project and launch, update, test, and debug in Docker containers on the desktop.
```

Comment:
maybe instead of 

> regardless of which language you use

we could broaden that to

> regardless of which technologies or language you use.
or 
> regardless of which runtime or language you use.
or
> regardless of which application stack you use.